website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Move from dbt Core to dbt Cloud - Getting started
+id: core-to-cloud-1
+description: "Learn how to move from dbt Core to dbt Cloud and what you need to get started."
+hoverSnippet: "Use this guide to learn how to move from dbt Core to dbt Cloud."
+icon: 'guides'
+time_to_complete: 'Total estimated time: 3-4 hours'
+hide_table_of_contents: true
+tags: ['Migration','dbt Core','dbt Cloud']
+level: 'Intermediate'
+recently_updated: true
+---
+## Introduction
+
+Moving from dbt Core to dbt Cloud streamlines analytics engineering workflows by allowing teams to develop, test, deploy, and explore data products using a single, fully managed service.
+
+- dbt Cloud is the fastest and most reliable way to deploy dbt. Develop, test, deploy, and explore data products using a single, fully managed service. It also supports: 
+  - The [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or command line with [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) for development
+  - [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) for consistent metrics
+  - Domain ownership of data with multi-project [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) setups
+  - [dbt Explorer](/docs/collaborate/explore-projects) for easier data discovery and understanding
+  
+  Learn more about [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features).
+
+- dbt Core is an open-source tool that enables data teams to transform data following analytics engineering best practices using a command line interface. It must be self-hosted and maintained.
+
+### What you'll learn
```

Comment:
Stylistically, we may want to avoid addressing the reader in the header. Maybe something like 
### Table of contents
### Agenda
or
### What is covered

Reply:
i thought so too but then i saw this in [unite test](https://docs.getdbt.com/docs/build/unit-tests#before-you-begin) and assumed it wasn't an issue?

Reply:
We should probably change that there too or get team consensus

Reply:
happy to add an issue to cover this! but it shouldn't be a blocker for this pr.

Reply:
added it to the docs sync: https://www.notion.so/dbtlabs/Docs-team-meeting-template-676bc701fd5b4a2b938879ef2482dbe8?pvs=4#a2cef8469f3547da8143fe71e708f899